### PR TITLE
Fix Spectral Normalization layer

### PR DESCRIPTION
**Pull request by markub3327**
Hi,

   spectral normalization layer is sensitive to initialized weights to zeroes. 
I fix it.

Thanks.

### CODE REVIEW

**Comment by fsx950223** [comment]
Any reference about it?

**Comment by markub3327** [comment]
```python
        self.discriminator = keras.Sequential(
            [
                keras.Input(shape=(64, 64, 3)),
                layers.GaussianNoise(0.1),

                layers.Conv2D(64, 7, padding="same", kernel_initializer=TruncatedNormal(stddev=0.02)),
                layers.LeakyReLU(alpha=0.2),
                layers.Dropout(0.3),

                layers.Conv2D(128, 4, strides=(2, 2), padding="same", kernel_initializer=TruncatedNormal(stddev=0.02)),
                layers.BatchNormalization(),
                layers.LeakyReLU(alpha=0.2),
                layers.Dropout(0.3),

                layers.Conv2D(128, 4, strides=(2, 2), padding="same", kernel_initializer=TruncatedNormal(stddev=0.02)),
                layers.BatchNormalization(),
                layers.LeakyReLU(alpha=0.2),
                layers.Dropout(0.3),

                layers.Conv2D(128, 4, strides=(2, 2), padding="same", kernel_initializer=TruncatedNormal(stddev=0.02)),
                layers.BatchNormalization(),
                layers.LeakyReLU(alpha=0.2),
                layers.Dropout(0.3),

                layers.Conv2D(128, 4, strides=(2, 2), padding="same", kernel_initializer=TruncatedNormal(stddev=0.02)),
                layers.BatchNormalization(),
                layers.LeakyReLU(alpha=0.2),
                layers.Dropout(0.3),

                layers.GlobalAveragePooling2D(),
                layers.Dropout(0.3),

                layers.Dense(1, kernel_initializer="zeros"),
            ],
            name="discriminator",
        )
```

**Comment by seanpmorgan** [comment]
Thank you for your contribution. We sincerely apologize for any delay in reviewing, but TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
